src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -22,6 +22,16 @@ When the HTTP request throws an `Exception`, Sentry's SDK associates this except
 
 For more information see our [SentryHttpClient integration](/platforms/dart/configuration/integrations/http-integration/#performance-monitoring-for-http-requests).
 
+### Dio Library Instrumentation
```

Comment:
It's a dup from the Dart integration, but it's important that Fluter users see it too, is there a way to not duplicate this?